connector/protobuf/src/main/scala/org/apache/spark/sql/protobuf/ProtobufDeserializer.scala

```diff
@@ -70,6 +69,15 @@ private[sql] class ProtobufDeserializer(
 
   def deserialize(data: Message): Option[InternalRow] = converter(data)
 
+  // JsonFormatter use to convert Any fields (if the option is enabled)
+  // This keeps original field names and does not include any extra whitespace in JSON.
+  // This is used to convert Any fields to json (if configured in Protobuf options).
+  // If the runtime type for Any field is not found in the registry, it throws an exception.
+  private val jsonPrinter = JsonFormat.printer
+    .omittingInsignificantWhitespace()
+    .preservingProtoFieldNames()
```

Comment:
@rangadi Are the names of Protobuf fields preserved for other types? I don't see any explicit use of this for other types.

Reply:
Yes, we do use field names as they appear in Protobuf definition. 
The setting here only applies to json deserialization. Without this, "event_name" field in example will appear as "eventName" in the JSON.

Reply:
@SandishKumarHN please approve this PR if it looks good. Planning to get this merged.